karma.conf.js

```diff
@@ -16,6 +16,7 @@ module.exports = function (config) {
             'karma-phantomjs-launcher',
             'karma-coverage'
         ],
+        usePolling: true,
```

Comment:
I was attempting to fix the build on the VSO hosted build agent.  No luck so I will remove this line.